### ISSUE TITLE
[ACCESSIBILITY] Dropdown menu not labeled correctly

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#4182]
**Describe the issue**
"Contact us button" is not the right label for this dropdown. This dropdown should be labeled as such like the "Appointments" screen "McKinley" dropdown button.  

**To Reproduce**
**Steps to reproduce the behavior:**
1. Using TalkBack
2. Go to 'Settings.'
3. Click on 'Contact us button.'

**Expected behavior**
The dropdown should be labeled as such like the "Appointments" screen "McKinley" dropdown button.  

**Actual behavior**
After gesturing past the close button the user hears "Contact us button" and user has no idea this is a dropdown menu with more options to gesture through as they then encounter Bug#4181 with weird sequence. 

**Screenshots**
![20240625_185736](https://github.com/rokwire/illinois-app/assets/82892873/1f04aefa-a782-478e-9fb8-9d440b7ee378)

**Testing Environment & Devices (please complete the following information):**
•	OS: Android Version 13
•	Device: Google Pixel 4a

**Additional context**
This should be applied to any dropdown menu used in the app.

**Comment by acovher2** [comment]
@TodorBachvarov Awesome!! Amazing turnaround time. I will retest as soon as I get that version. 

Kindly, Amanda

**Comment by acovher2** [comment]
@TodorBachvarov Can you apply this fix to the Assistant dropdown as well? or would you like a separate bug for that?

**Comment by TodorBachvarov** [comment]
@acovher2, Fixed for Assistant in version 6.0.20

**Comment by acovher2** [comment]
Verified fixed in 6.0.32 
 Although I found more assistant issues for the focus landing on the screen and mic s2t
